src/Page/Home.elm

```diff
@@ -80,115 +55,31 @@ viewHero { enabledSections } modal =
             [ h2 [ class "h1" ]
                 [ text "Calculez le coût environnemental de vos produits" ]
             , div [ class "fs-5 mt-3 mb-5" ]
-                [ """Ecobalyse permet de comprendre et de calculer le coût environnemental des produits
-                     distribués en France. La méthodologie présentée est soumise à [concertation]({url})."""
-                    |> String.replace "{url}" Env.communityUrl
+                [ """Ecobalyse permet de comprendre et d’exprimer les impacts environnementaux des produits distribués en France par le calcul d’un coût en points d’impact\u{202F}: le coût environnemental. Découvrez nos outils et notre périmètre d’action\u{202F}!"""
                     |> Markdown.simple []
                 ]
             , div [ class "d-flex flex-column flex-sm-row gap-3 mb-4" ]
-                [ if enabledSections.food then
-                    button [ class "btn btn-lg btn-primary", onClick OpenCalculatorPickerModal ]
-                        [ text "Lancer le calculateur" ]
+                [ a [ class "btn btn-lg btn-primary d-flex align-items-center justify-content-center", Route.href Route.TextileSimulatorHome ]
+                    [ text "Calculer l’impact d’un vêtement" ]
+                , if enabledSections.food then
+                    a [ class "btn btn-lg btn-outline-primary", Route.href Route.FoodBuilderHome ]
+                        [ text "Calculer l’impact de l’alimentation", br [] [], Html.cite [ class "fw-normal fs-7 d-block" ] [ text "Méthodologie en concertation" ] ]
```

Comment:
Nit: maybe spread this long line over multiple ones for legibility. also, we should use a `div` or `p` (block element) instead of `br`